test/IonParserBinaryRaw.ts

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+import {assert} from 'chai';
+import * as ion from '../src/IonTests';
+
+/**
+ * UInt
+ *
+ * Spec: http://amzn.github.io/ion-docs/docs/binary.html#uint-and-int-fields
+ */
```

Comment:
I don't quite understand the context of this comment...

Reply:
Each set of tests is grouped by ion datatype. The comment is intended to be a section header and isn't associated with the function that follows. Do you have a suggestion as to how I can make that clearer?

Reply:
I would state it thusly: "Tests for UInt encoding..."